### PR TITLE
Add maven to dependabot

### PR DESCRIPTION
**Pull request by sebastiaanspeck**

### CODE REVIEW

**Comment by StevenLooman** [comment]
Thanks, but let change it to monthly.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ updates:
     directory: "/"
     schedule:
       interval: weekly
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: weekly
```
Comment (sebastiaanspeck) [inline .github/dependabot.yml]:
```suggestion
      interval: monthly
```

**Comment by sebastiaanspeck** [comment]
> Thanks, but let change it to monthly.

Done!

**Comment by StevenLooman** [comment]
Thanks!

**Pull request merged**